app/subscriber/src/features/my-reports/view/components/UserContentForm.tsx

```diff
@@ -58,6 +61,7 @@ export const UserContentForm: React.FC<IUserContentFormProps> = ({
           name={`headline`}
           label="Headline"
           rows={1}
+          error={errors && errors.headline && errors.headline}
```

Comment:
An odd syntax.  Why not just supply `errors?.headline`?

Reply:
totally ... fixing it ...